.github/workflows/ci-workflow.yml

```
@@ -33,3 +33,19 @@ jobs:
         run: npm clean-install
       - name: Run integration tests
         run: npm run integration-test
+  code_coverage:
+    runs-on: ubuntu-latest
```

Je pense qu'il serait plus utile de lancer ce job après avoir passé les tests d'intégration : 

```suggestion
    needs: integration-tests
    runs-on: ubuntu-latest
```

---

ok corrigé